identifiers/country-gb.csv

```diff
@@ -1,665 +1,181 @@
-ocd-division/country:gb/part:sct/ed:rutherglen_and_hamilton_west,Rutherglen and Hamilton West
-ocd-division/country:gb/part:sct/ed:stirling,Stirling
-ocd-division/country:gb/part:sct/ed:west_aberdeenshire_and_kincardine,West Aberdeenshire and Kincardine
-ocd-division/country:gb/part:sct/ed:west_dunbartonshire,West Dunbartonshire
-ocd-division/country:gb/part:wls,Wales
-ocd-division/country:gb/part:wls/ed:aberavon,Aberavon
-ocd-division/country:gb/part:wls/ed:aberconwy,Aberconwy
-ocd-division/country:gb/part:wls/ed:alyn_and_deeside,Alyn and Deeside
-ocd-division/country:gb/part:wls/ed:arfon,Arfon
-ocd-division/country:gb/part:wls/ed:blaenau_gwent,Blaenau Gwent
-ocd-division/country:gb/part:wls/ed:brecon_and_radnorshire,Brecon and Radnorshire
-ocd-division/country:gb/part:wls/ed:bridgend,Bridgend
-ocd-division/country:gb/part:wls/ed:caerphilly,Caerphilly
-ocd-division/country:gb/part:wls/ed:cardiff_central,Cardiff Central
-ocd-division/country:gb/part:wls/ed:cardiff_north,Cardiff North
-ocd-division/country:gb/part:wls/ed:cardiff_south_and_penarth,Cardiff South and Penarth
-ocd-division/country:gb/part:wls/ed:cardiff_west,Cardiff West
-ocd-division/country:gb/part:wls/ed:carmarthen_east_and_dinefwr,Carmarthen East and Dinefwr
-ocd-division/country:gb/part:wls/ed:carmarthen_west_and_south_pembrokeshire,Carmarthen West and South Pembrokeshire
-ocd-division/country:gb/part:wls/ed:ceredigion,Ceredigion
-ocd-division/country:gb/part:wls/ed:clwyd_south,Clwyd South
-ocd-division/country:gb/part:wls/ed:clwyd_west,Clwyd West
-ocd-division/country:gb/part:wls/ed:cynon_valley,Cynon Valley
-ocd-division/country:gb/part:wls/ed:delyn,Delyn
-ocd-division/country:gb/part:wls/ed:dwyfor_meirionnydd,Dwyfor Meirionnydd
-ocd-division/country:gb/part:wls/ed:gower,Gower
-ocd-division/country:gb/part:wls/ed:islwyn,Islwyn
-ocd-division/country:gb/part:wls/ed:llanelli,Llanelli
-ocd-division/country:gb/part:wls/ed:merthyr_tydfil_and_rhymney,Merthyr Tydfil and Rhymney
-ocd-division/country:gb/part:wls/ed:monmouth,Monmouth
-ocd-division/country:gb/part:wls/ed:montgomeryshire,Montgomeryshire
-ocd-division/country:gb/part:wls/ed:neath,Neath
-ocd-division/country:gb/part:wls/ed:newport_east,Newport East
-ocd-division/country:gb/part:wls/ed:newport_west,Newport West
-ocd-division/country:gb/part:wls/ed:ogmore,Ogmore
-ocd-division/country:gb/part:wls/ed:pontypridd,Pontypridd
-ocd-division/country:gb/part:wls/ed:preseli_pembrokeshire,Preseli Pembrokeshire
-ocd-division/country:gb/part:wls/ed:rhondda,Rhondda
-ocd-division/country:gb/part:wls/ed:swansea_east,Swansea East
-ocd-division/country:gb/part:wls/ed:swansea_west,Swansea West
-ocd-division/country:gb/part:wls/ed:torfaen,Torfaen
-ocd-division/country:gb/part:wls/ed:vale_of_clwyd,Vale of Clwyd
-ocd-division/country:gb/part:wls/ed:vale_of_glamorgan,Vale of Glamorgan
-ocd-division/country:gb/part:wls/ed:wrexham,Wrexham
-ocd-division/country:gb/part:wls/ed:ynys_môn,Ynys Môn
+id,name,gss_code

```

Comment:
Did something go wrong in the compile step? There are far fewer OCDIDs than before. I think all UK Parliament constituencies are missing.

Reply:
Good spot. It must have, but I'm not sure why. I re-ran `./scripts/compile.py gb` + fixed in 140a525